moveit_core/constraint_samplers/src/constraint_sampler_tools.cpp

```diff
@@ -103,20 +110,20 @@ void constraint_samplers::visualizeDistribution(const ConstraintSamplerPtr& samp
     if (!sampler->sample(ks))
       continue;
     const Eigen::Vector3d& pos = ks.getGlobalLinkTransform(lm).translation();
-    visualization_msgs::Marker mk;
-    mk.header.stamp = ros::Time::now();
+    visualization_msgs::msg::Marker mk;
+    mk.header.stamp = stamp;
```

Comment:
This could be set directly from `rclcpp::Clock().now()`

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/082b80f6633611ad8f8a02b44733b5e544f5d232